gen/compile_proto.jl

```diff
@@ -21,63 +21,47 @@ pbpath =dirname(dirname(PosixPath(pathof(ProtoBuf))))/p"gen"
 cur_path = cwd()
 TBL_root = dirname(cur_path)
 
-src_dir = cur_path/"proto"
-out_dir   = cur_path/"protojl"
+# src_dir = cur_path/"proto"
+src_dir = PosixPath(".")/"proto"
```

Comment:
Since this only runs **once** when building the package (specifically, when updating ProtoBuf) I don't think this should pose a problem. In particular there is nothing windows specific here (the generated files are commited and this has no reason to run under windows).

However if any Unit test fails under windows, then we can see about this.

Reply:
In principle it should work either way but ProtoBuf 1.0.0 code generation is annoying, so if at all possible I would like to avoid repeating this.